compiler/util-klib/src/org/jetbrains/kotlin/library/SearchPathResolver.kt

```diff
@@ -139,7 +139,7 @@ abstract class KotlinLibrarySearchPathResolver<L : KotlinLibrary>(
             val matching = fileSequence
                 .filterOutPre_1_4_libraries()
                 .flatMap { libraryComponentBuilder(it, isDefaultLink).asSequence() }
-                .map { it.takeIf { libraryMatch(it, unresolved) } }
+                //.map { it.takeIf { libraryMatch(it, unresolved) } }
```

Comment:
I need atomicfu klib as the compile dependency for the plugin tests. This dependency is provided in the [atomicfuClasspath](https://github.com/JetBrains/kotlin/blob/19610be7c35dcf39b3db4a076bad6891c30fc043/plugins/atomicfu/atomicfu-compiler/build.gradle.kts#L48) - the atomicfu-js version `0.14.3-1.4-M1`, that is built with `1.4-M1`. The problem here is that kotlin abi version  of this dependency is 1.4.0 while the CURRENT KotlinAbiVersion is 1.4.1, so here I get mismatch and the dependency is not used.

How is the abi version provided for the library?